docs/gcloud-config.rst

```diff
@@ -0,0 +1,55 @@
+Configuration
+*************
+
+Overview
+========
+
+- Use service :class:`Client <gcloud.client.Client>` objects to configure
```

Comment:
Since only package-specific subclasses of `Client` are used, it's not worth mentioning the base class. Just use the word "client" instead.